README.md

```diff
@@ -45,6 +45,19 @@ Check out these [code examples](examples) in JavaScript and TypeScript to get up
 
 If your environment requires SSL decryption, you can set the path to CA bundle in the env var `TWILIO_CA_BUNDLE`.
 
+### Client Initialization
+If you invoke any V2010 operations without specifying an account SID, `twilio-node` will automatically use the `TWILIO_ACCOUNT_SID` value that the client was initialized with. This is useful for when you'd like to, for example, fetch resources for your main account but also your subaccount. See below:
+
+```javascript
+var accountSid = process.env.TWILIO_ACCOUNT_SID; // Your Account SID from www.twilio.com/console
+var authToken = process.env.TWILIO_AUTH_TOKEN;   // Your Auth Token from www.twilio.com/console
+var subAccountSid = process.env.TWILIO_ACCOUNT_SUBACCOUNT_SID; // Your Subacount SID from www.twilio.com/console
```

Comment:
nit: `subaccount` should be 1 word (lowercase `a`). Also missing a `c` in `acount`.